sunpy/net/dataretriever/sources/eve.py

```diff
@@ -0,0 +1,105 @@
+# Author: Rishabh Sharma <rishabh.sharma.gunner@gmail.com>
+# This module was developed under funding by
+# Google Summer of Code 2014
+
+from __future__ import absolute_import, division, print_function
+
+from sunpy.extern.six.moves.urllib.parse import urljoin
+
+from ..client import GenericClient
+
+
+__all__ = ['EVEClient']
+
+
+class EVEClient(GenericClient):
```

Comment:
In the future can this be expanded to download other level of data or will we need another client?  If the latter, we should change the class name to something like EVESpWxClient consisent with our discussion on the weekly meeting of Nov. 30th 2016.